Classes/Event/EnrichFileDataEvent.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the "headless" Extension for TYPO3 CMS.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.md file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace FriendsOfTYPO3\Headless\Event;
+
+use TYPO3\CMS\Core\Resource\FileInterface;
+
+final class EnrichFileDataEvent
+{
+    private ?FileInterface $fileReference;
```

Comment:
I missed first time, why $fileReference is nullable? in constructor you force to be object & not allow for null